index.html

```diff
@@ -353,6 +354,14 @@
             </li>
           </ul>
         </section>
+        <section id="httpGETLDPRS">
+          <h2><a>LDP-RS</a>s</h2>
+          <p>
+            <code>GET</code> requests that include a <code>Prefer</code> header to any <a>LDP-RS</a> MUST
```

Comment:
👍  that is much clearer — I've just pushed a new commit with this updated language.